<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServerWindow</class>
 <widget class="QMainWindow" name="TcpServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCP服务器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>本地地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_comboLoalAddr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>0.0.0.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>本地端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_editLocalPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="m_editOutput">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="m_editSend"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnSend">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actListen"/>
   <addaction name="m_actClose"/>
   <addaction name="m_actClear"/>
   <addaction name="separator"/>
   <addaction name="m_actExit"/>
  </widget>
  <action name="m_actListen">
   <property name="icon">
    <iconset resource="TcpServer.qrc">
     <normaloff>:/images/listen.bmp</normaloff>:/images/listen.bmp</iconset>
   </property>
   <property name="text">
    <string>监听</string>
   </property>
   <property name="toolTip">
    <string>监听</string>
   </property>
  </action>
  <action name="m_actClose">
   <property name="icon">
    <iconset resource="TcpServer.qrc">
     <normaloff>:/images/close.bmp</normaloff>:/images/close.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="m_actClear">
   <property name="icon">
    <iconset resource="TcpServer.qrc">
     <normaloff>:/images/clear.bmp</normaloff>:/images/clear.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="m_actExit">
   <property name="icon">
    <iconset resource="TcpServer.qrc">
     <normaloff>:/images/exit.bmp</normaloff>:/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="TcpServer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_actExit</sender>
   <signal>triggered()</signal>
   <receiver>TcpServerWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
